core/shared/src/main/scala/cats/parse/Parser.scala

```diff
@@ -388,6 +388,21 @@ sealed abstract class Parser1[+A] extends Parser[A] {
   def rep1(min: Int): Parser1[NonEmptyList[A]] =
     Parser.rep1(this, min = min)
 
+  /** Use this parser to parse between values.
+    *
+    * Parses `b` followed by `this` and `c`.
+    * Returns only the values extracted by `this` parser.
+    */
+  def between[B, C](b: Parser[B], c: Parser[C]): Parser[A] =
```

Comment:
Can we used `Parser[Any]` here and leverage covariance? That way the signature is simpler (no type params).

Reply:
gotcha! yeah that makes sense